SPECS/maven/maven3.spec

```diff
@@ -39,6 +40,16 @@ Conflicts:      maven
 %description
 Maven is a software project management and comprehension tool. Based on the concept of a project object model (POM). Maven can manage a project's build, reporting and documentation from a central piece of information.
 
+%package openjdk11
+Summary:        MSOpenJDK 11 binding for Maven
```

Comment:
Is there a reason for making this change?

Reply:
This is required for `xmvn` package, however it uses the `maven` not `maven3` so technically not. But I was adding it just because that is something that I have changed in maven.